### ISSUE TITLE
[UI] multi partitioned assets - fix highlighted event

### ISSUE THREAD

**Issue by geoHeil** [dagster-io/dagster#18959]
### Dagster version

1.5

### What's the issue?

when using multi partitioned assets the green dots of the already materialized data are not changing correctly when switching tabs. I.e.

 the correct visualization would. be

![Bildschirmfoto 2023-12-30 um 17 28 15](https://github.com/dagster-io/dagster/assets/1694964/015266a4-a3b2-4906-87e2-a32c2b2e0378)

but when switching back and forth between the tabs i.e. events / plots and back to partitions what we get is:
![Bildschirmfoto 2023-12-30 um 17 28 25](https://github.com/dagster-io/dagster/assets/1694964/fdb059bc-d30e-4ae7-9183-1f109700fa6e)

assets for all partitions are highlighted.

Only when clicking some primary partitions the right green lights are flashing again

### What did you expect to happen?

to see the right green indicators.

### How to reproduce?

Find the code I used for the specification of the partition below:

```python
import pandas as pd

def get_cc_main_partitions():
    return pd.read_html("https://data.commoncrawl.org/crawl-data/index.html")[0][
        "ID/Location"
    ].values.tolist()

seednodes_def = DynamicPartitionsDefinition(name="seednodes")

cc_partitions_def = MultiPartitionsDefinition(
    {
        "crawl": StaticPartitionsDefinition(get_cc_main_partitions()),
        "a_seednodes": seednodes_def,

    }
)
```

as you can see it is a combination of DynamicPartitionsDefinition and StaticPartitionsDefinition.

### Deployment type

Local

### Message from the maintainers

Impacted by this issue? Give it a 👍! We factor engagement into prioritization.

**Comment by bengotow** [comment]
Hey @geoHeil - I believe this is fixed in the latest release (1.5.13). In the screenshots you provided, the correct dots are present but they're wrapping on to the next line, which was fixed.

The behavior of this UI for the second dimension of a 2D partitioned asset is a bit subtle though.

In your second screenshot, no partition dimensions are selected, so "CC-MAIN-2023-50" is showing both a gray dot AND a green dot (they should be side by side but are wrapping.) We consider that the correct behavior, because for `sample_seeds_1` that partition is materialized and for `p2` it is missing.  When you select one of the first dimension values (the leftmost column), the status dots for the second dimension change to reflect just the partitions within that selection. So if you choose "sample_seeds_1", you should see a green dot, and if you choose "p2" you should see a gray dot.

Hope that helps! Let me know if you're still seeing incorrect values after the 1.5.13 update.

**Comment by geoHeil** [comment]
I am now on 1.5.14
<img width="663" alt="Bildschirmfoto 2024-01-08 um 20 54 42" src="https://github.com/dagster-io/dagster/assets/1694964/9dca513b-4dbf-45a3-b833-fb6e0be18a9c">

this already looks much better - thanks

seems to be fixed now great
